.github/workflows/ingest-genbank.yml

```diff
@@ -0,0 +1,33 @@
+name: Ingest 2019-nCov/SARS-CoV-2 data from GenBank for nextstrain.org/ncov
+
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # "At minute 0 past every 4th hour"
+    - cron:  '0 */4 * * *'
```

Comment:
This should be `*/6` to reflect the latest changes on master (and is what ingest-gisaid.yml uses because of the rebase).